public/app/features/browse-dashboards/components/NewProvisionedFolderForm.tsx

```diff
@@ -77,21 +75,40 @@ export function NewProvisionedFolderForm({ onSubmit, onCancel, parentFolder }: P
     const appEvents = getAppEvents();
     if (request.isSuccess) {
       onSubmit();
-      if (workflow === 'branch' && prLink) {
-        locationService.partial({ prLink });
-      } else {
-        appEvents.publish({
-          type: AppEvents.alertSuccess.name,
-          payload: ['Folder created successfully'],
-        });
+
+      appEvents.publish({
+        type: AppEvents.alertSuccess.name,
+        payload: ['Folder created successfully'],
+      });
+
+      const folder = request.data.resource?.upsert as Resource;
```

Comment:
We need to get these types fixed on backend somehow, if this is a Resource then I'm not sure why it's returned as Unstructured.